src/examples/SingleSignOn/SingleSignOn.code.js

```diff
@@ -0,0 +1,25 @@
+import { indent } from "../../utils";
+// eslint-disable-next-line import/no-unresolved, import/extensions
+import { SingleSignOn } from "!!./SingleSignOn";
+
+export const singleSignOnCode = `
+<head>
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  <script src="https://static.qgov.net.au/formio-qld/v1/v1.x.x-latest/formio-script.prod.min.js"></script>
```

Comment:
why is there 'prod' in the name?

Reply:
formio-script.prod.min.js will load the scripts defined in src/helpers/FormioScript/index.prod.js
formio-script.dev.min.js will load the scripts defined in src/helpers/FormioScript/index.dev.js
etc.

Reply:
maybe the prod one can be renamed to `formio-script.prod.min.js`

Reply:
if you can, that would be great.